ddp-workspace/projects/ddp-sdk/src/lib/components/user/manageParticipants.component.ts

```diff
@@ -21,7 +23,7 @@ import { tap, mergeMap } from 'rxjs/operators';
     <mat-list style="margin-bottom:10px">
         <h3 mat-subheader translate>SDK.ManageParticipants.ParticipantsList</h3>
         <mat-list-item *ngFor="let participant of participants" class="card-1">
-            {{ participant.alias }}
+            {{ participant.userProfile.firstName }} {{ participant.userProfile.lastName }}
```

Comment:
As Marco mentioned in another PR:
"Can we always expect that userProfile property will be set? (I don't know the answer).
If not sure or don't know then try to be defend for undefined or null condition here."

Reply:
The only way to create a participant from frontend is to call `addParticipant` method on `GovernedParticipantsServiceAgent`. Request issued to an associated endpoint will create a participant which will always have an associated profile (therefore `userProfile` field won't be `null | undefined`).